selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/workspaces/CheckStopStartWsTest.java

```diff
@@ -43,16 +42,15 @@ public void setUp() throws Exception {
   @Test
   public void checkStopStartWorkspaceTest() {
     projectExplorer.waitProjectExplorer();
-    terminal.waitTerminalTab(LOADER_TIMEOUT_SEC);
-    loader.waitOnClosed();
+    terminal.waitTerminalTab(EXPECTED_MESS_IN_CONSOLE_SEC);
+    menu.waitMenuItemIsEnabled(PROFILE_MENU);
```

Comment:
This verification slightly duplicate another one in test [CreateWorkspaceOnDashboardTest](https://github.com/eclipse/che/blob/master/selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/workspaces/CreateWorkspaceOnDashboardTest.java#L86).
To avoid duplication and to have one place to change verification in future, I suggest to move the next commands from the test [CreateWorkspaceOnDashboardTest](https://raw.githubusercontent.com/eclipse/che/master/selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/workspaces/CreateWorkspaceOnDashboardTest.java) to separate method `Ide::waitOpenedWorkspaceIsReadyToUse()` and use it instead:
```java
    projectExplorer.waitProjectExplorer(PREPARING_WS_TIMEOUT_SEC); // we need long timeout for OSIO
    terminal.waitTerminalTab(PREPARING_WS_TIMEOUT_SEC); // we need long timeout for OSIO
    menu.waitMenuItemIsEnabled(WORKSPACE);
```